integrations/gradle-bloop/src/test/scala/bloop/integrations/gradle/ConfigGenerationSuite.scala

```diff
@@ -47,6 +47,234 @@ abstract class ConfigGenerationSuite extends BaseConfigSuite {
   private val testProjectDir_ = new TemporaryFolder()
   @Rule def testProjectDir: TemporaryFolder = testProjectDir_
 
+  @Test def worksWithAndroidPlugin(): Unit = {
+    // This version has to match with Dependencies#gradleAndroidPluginVersion.  Currently can't test multiple versions.
+    worksWithAndroidPlugin("4.1.0")
+  }
+
+  private def worksWithAndroidPlugin(androidToolsVersion: String): Unit = {
+    if (gradleVersion >= "6.1.1") {
+      val buildSettings = testProjectDir.newFile("settings.gradle")
+      val buildDirA = testProjectDir.newFolder("a")
+      testProjectDir.newFolder("a", "src", "main", "java")
+      testProjectDir.newFolder("a", "src", "androidTest", "java")
+      val buildDirB = testProjectDir.newFolder("b")
+      testProjectDir.newFolder("b", "src", "main", "java")
+      testProjectDir.newFolder("b", "src", "androidTest", "java")
+      val buildDirC = testProjectDir.newFolder("c")
+      testProjectDir.newFolder("c", "src", "main", "java")
+      testProjectDir.newFolder("c", "src", "androidTest", "java")
+      val buildFileA = new File(buildDirA, "build.gradle")
+      val buildFileB = new File(buildDirB, "build.gradle")
+      val buildFileC = new File(buildDirC, "build.gradle")
+
+      writeBuildScript(
+        buildFileA,
+        s"""
+           |buildscript {
+           |  repositories {
+           |    google()
+           |    jcenter()
```

Comment:
```suggestion
           | 
```
Is the repo still available? Probably best to remove it.

Reply:
Removed